docs/fr-FR/newbs_building_firmware_configurator.md

```diff
@@ -0,0 +1,105 @@
+# Configurateur de QMK
+
+Le [Configurateur de QMK](https://config.qmk.fm) est une interface graphique en ligne permettant de générer des fichiers "hex" du firmware de QMK.
+
+?> **S'il vous plaît, suivez les étapes suivantes dans l'ordre.**
+
+Regardez le [Tutoriel vidéo](https://youtu.be/tx54jkRC9ZY)
+
+Le configurateur de QMK fonctionne mieux avec Chrome et Firefox.
+
+!> **Les fichiers d'autres outils, tels que KLE ou kbfirmware ne seront pas compatibles avec le configurateur QMK. Ne les chargez pas, ne les importez pas. Le configurateur QMK est un outil DIFFERENT.**
+
+## Sélectionner votre clavier
+
+Cliquez la boîte déroulante et sélectionnez le clavier pour lequel vous voulez créer une keymap.
+
+?> Si votre clavier a plusieurs verisons, faites attention à utiliser la version correcte.
+
+Je vais le répéter, parce que c'est important
+
+!> **FAITES ATTENTION A UTILISER LA BONNE VERSION !**
+
+Si votre clavier est annoncé comme fonctionnant grâce à QMK mais n'est pas dans la liste, il y a des chances que le développeur ne l'ait pas encore fait, ou que nous n'avons pas encore eu le temps de le merger. Ajoutez un problème (issue) sur [qmk_firmware](https://github.com/qmk/qmk_firmware/issues) demandant le support de votre clavier, s'il n'y a pas de [Pull Request](https://github.com/qmk/qmk_firmware/pulls?q=is%3Aopen+is%3Apr+label%3Akeyboard) ouvert pour lui. Il y a aussi des clavier alimentés par QMK qui sont sur le compte GitHub du fabriquant, il est bon de le vérifier aussi.
```

Comment:
`un problème (issue)` => `une issue`
no need to translate it.

Reply:
Translated elsewhere by "ticket" wich seemed better. What do you think?

Reply:
I'd prefer to stay on `issue` because it's the most common usage. It would be weird to translate it.

Reply:
agreed, used issue